Source/WebKit/UIProcess/mac/WKViewLayoutStrategy.mm

```diff
@@ -136,9 +136,6 @@ - (void)didEndLiveResize
 
 - (void)didChangeFrameSize
 {
-    if ([self frameSizeUpdatesDisabled])
```

Comment:
You should remove all the code out from here, too.

Reply:
yup I know. This is just a draft PR for now until I fix everything up